src/status_im2/subs/communities.cljs

```diff
@@ -71,16 +71,16 @@
           (sort-by #(visibility-status-utils/visibility-status-order (get % 0)))))))
 
 (re-frame/reg-sub
- :communities/featured-communities
- :<- [:communities]
- (fn [communities]
-   (vals communities)))
+ :communities/featured-contract-communities
+ :<- [:contract-communities]
+ (fn [cc]
+   (sort-by :name (vals (:featured cc)))))
 
 (re-frame/reg-sub
- :communities/sorted-communities
- :<- [:communities]
- (fn [communities]
-   (sort-by :name (vals communities))))
+ :communities/other-contract-communities
+ :<- [:contract-communities]
+ (fn [cc]
```

Comment:
same here `contract-communities` , why are you afraid of readable names ? :)

Reply:
Thanks @flexsurfer! I like using short abbreviations, like the typical clojure `k v m s` etc. as long as there is a line early in the function where a reader can look up the full name (here `:contact-communities` is right above `cc`). All those long names make the code longer and harder to read, IMO. No worries though, I'll change it to the longer name!